spec/services/checksum_validator_spec.rb

```diff
@@ -203,24 +203,60 @@
         expect(results).to receive(:add_result).with(
           AuditResults::INVALID_MANIFEST, manifest_file_path: 'spec/fixtures/checksum_root01/moab_storage_trunk/xx/444/xx/4444/xx444xx4444/v0001/manifests/signatureCatalog.xml'
         )
-        cv.validate_signature_catalog
+        cv.send(:validate_signature_catalog_listing)
       end
     end
   end
 
-  context '#validate_checksum' do
+  context '#flag_unexpected_data_content_files' do
     let(:druid) { 'bj102hs9687' }
-    let(:cv) { described_class.new(druid, 'fixture_sr1') }
+    let(:cv) { described_class.new(druid, endpoint_name) }
 
-    it 'returns a positive result for a druid that passes validation' do
-      cv.validate_checksum
-      expect(cv.checksum_results.result_array.first).to have_key(:moab_checksum_valid)
+    it 'calls validate_against_signature_catalog on each of the data_content_files' do
```

Comment:
so this might need to show it's called:
- for a metadata file
- for a top level content file
- for a file in a content subdir 
- ...